### DISCUSSION TITLE
Blacklist GPO entry not working

### DISCUSSION THREAD

**Discussion by JasonCioffi** [Romanitho/Winget-AutoUpdate#613] [Q&A] [answered]
Hi,

Program ID 'Webroot.SecureAnywhere', available in the winget repository, is a different version than the one I'm using (Webroot Endpoint Protection) so I can't install it or update it through winget.  After installing Webroot Endpoint Protection manually I noticed it assumes program ID 'Webroot.SecureAnywhere' so when the WAU update interval kicks in it attempts to update the program and ultimately errors out.  I thought I could avoid the error by adding Webroot.SecureAnywhere to the blacklist GPO, but WAU keeps attempting to update the app.  I confirmed with gpresult the blacklist GPO is applying successfully so I'm not sure why WAU is still trying to update Webroot.

I'm assuming the blacklist GPO entries update a file or the registry.  Can someone please advise what I should look at to confirm the GPO is updating whatever needs to be updated to tell WAU to skip its attempt to update that program?  Or maybe there's something else I'm missing?

Thanks in advance for your replies.

**Comment by KnifMelti** [comment, marked as answer]
Have you enabled and filled in ...external Path: `GPO` (and configured the GPO Blacklist):
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/4ed2502e-b63a-48f9-8602-7fbf974c983c)

The llist is written to and imported from under the standard path for policies:
`HKLM:\SOFTWARE\Policies\Romanitho\Winget-AutoUpdate\BlackList`

The GPO import runs once a day via task scheduler:
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/545a087b-2cb9-4b70-a250-af87d1025520)
(I have disabled mine, running it everytime WAU runs via **_WAU-mods.ps1**)

**Comment by JasonCioffi** [comment]
Thank you for the reply.

I did not have the "Get Black/White list from external path" gpo enabled.  I had the other two GPO settings configured per your screenshot already and I saw the blacklist entries in my registry.  I enabled "Get Black/White list..." and set the path to 'GPO'.  I forced the Winget-AutoUpdate-Policies task to run and then forced the Winget-Autoupdate tak to run.  WAU again attempted to update Webroot.  

Next, I issued a gpupdate /force and re-ran the two tasks and viola! no more Webroot update error!  Thanks again for your help.  I'll mark as answer.
